### ISSUE TITLE
Don't log meta tools version if there is no o2r-meta version file

### ISSUE THREAD

**Issue by LukasLohoff** [o2r-project/o2r-loader#29]
Loader doesn't start when the file "../o2r-meta/version" does not exist.

```
/home/.../o2r/o2r-loader/index.js:196
     debug('meta tools version: %s', data.trim());
                                         ^

TypeError: Cannot read property 'trim' of undefined
   at ReadFileContext.callback (/home/.../o2r/o2r-loader/index.js:196:43)
   at FSReqWrap.readFileAfterOpen [as oncomplete] (fs.js:366:13)
```

**Comment by LukasLohoff** [comment]
Next line is also missing a semicolon

**Comment by LukasLohoff** [comment]
fixed in #31
